### ISSUE TITLE
Temperature / Humidity from Shelly Addon

### ISSUE THREAD

**Issue by Ben19851337** [alexryd/homebridge-shelly-ng#106]
Hi, is there any possibility to get the temp. and humidity from the shelly addon to homekit?

Ben

**Comment by Tom-236** [comment]
Yes please, I also would be very interested for the Addon plus (2nd generation).

Best regards,

Tom

**Comment by Maverick5269-lab** [comment]
Hi,

I had that question too.
There is any possibility to have Temp reading from a Shelly 1 Plus with add-on? 


Thanks
Best Regards
